syntax = "proto3";
import "Rectangle.proto";
import "Point.proto";
package streetgangpb;


message GetSceneResponse
{
	uint64 world_id = 1;
	Rectangle rect = 2;
	repeated Point items = 3;
}
